client/settings/settings-manager/index.js

```diff
@@ -110,9 +111,23 @@ const DepositsDescription = () => {
 };
 
 const FraudProtectionDescription = () => {
+	const { isLoading } = useSettings();
+
+	useLayoutEffect( () => {
+		const { anchor } = getQuery();
+
+		if ( ! isLoading && anchor ) {
+			document
+				.querySelector( decodeURIComponent( anchor ) )
+				.scrollIntoView( { behavior: 'smooth' } );
```

Comment:
To prevent errors in case the anchor selector is invalid or the element doesn't exist, we can add a `?` before the `.scrollIntoView`, like this:
```js
document
  .querySelector( decodeURIComponent( anchor ) )
  ?.scrollIntoView( { behavior: 'smooth' } );
```

Reply:
Good catch! I've fixed this in [3198f43](https://github.com/Automattic/woocommerce-payments/pull/5611/commits/3198f439f55fd25e710d2fb3007d13f8843f3203).